server/athenian/api/internal/settings.py

```diff
@@ -702,23 +713,48 @@ async def list_release_matches(
         """
         if repos is None:
             repos = await get_account_repositories(self._account, True, self._sdb)
-        repos = set(repos).union(coerce_prefixed_logical_repos(repos).keys())
+
+        # collect also physical repositories for requested logical repositories
+        repos_to_collect = set(repos).union(coerce_prefixed_logical_repos(repos).keys())
+
+        repo_names_to_collect: list[RepositoryName] = [
+            RepositoryName.from_prefixed(repo) for repo in repos_to_collect
+        ]
+        repo_names_to_collect_set = set(repo_names_to_collect)
+
+        physical_unprefixed = {r.unprefixed_physical for r in repo_names_to_collect}
+        name_to_node = self._prefixer.repo_name_to_node.get
+        all_repo_ids = [
+            repo_id for unprefixed in physical_unprefixed if (repo_id := name_to_node(unprefixed))
+        ]
```

Comment:
```suggestion
        name_to_node = self._prefixer.repo_name_to_node.get
        all_repo_ids = [
            repo_id for r in repo_names_to_collect if (repo_id := name_to_node(r.unprefixed_physical))
        ]
```

Reply:
done, used set for all_repos_ids to avoid duplicates in query `WHERE repo_id IN (...`